include/rocksdb/db.h

```diff
@@ -1628,16 +1628,23 @@ class DB {
   //  s = db->SetOptions(cfh, {{"block_based_table_factory",
   //                            "{prepopulate_block_cache=kDisable;}"}});
   virtual Status SetOptions(
-      ColumnFamilyHandle* /*column_family*/,
-      const std::unordered_map<std::string, std::string>& /*opts_map*/) {
-    return Status::NotSupported("Not implemented");
+      ColumnFamilyHandle* column_family,
+      const std::unordered_map<std::string, std::string>& opts_map) {
+    return SetOptions(std::vector<ColumnFamilyHandle*>{column_family},
+                      opts_map);
   }
   // Shortcut for SetOptions on the default column family handle.
   virtual Status SetOptions(
       const std::unordered_map<std::string, std::string>& new_options) {
     return SetOptions(DefaultColumnFamily(), new_options);
   }
 
+  virtual Status SetOptions(
```

Comment:
Although I believe this satisfies the motivating need for this enhancement (apply one set of options to N CFs) I think we should go ahead and generalize all the way to N sets of options applied correspondingly to N CFs--in addition to this 1-to-N overload. That way no one finds their hands tied (again) with repeated SetOptions and we can knock it out while it's fresh on mind.